js/modules/loki_public_chat_api.js

```diff
@@ -555,17 +582,79 @@ class LokiPublicChannelAPI {
           adnMessage.annotations.length !== 0
         ) {
           const noteValue = adnMessage.annotations[0].value;
-          ({ timestamp, quote } = noteValue);
+          ({ timestamp } = noteValue);
 
-          if (quote) {
-            quote.attachments = [];
+          if (noteValue.quote) {
+            ({ quote } = noteValue);
+            if (quote) {
+              quote.attachments = [];
+            }
           }
 
-          // if user doesn't have a name set, fallback to annotation
-          // pubkeys are already there in v1 (first release)
-          if (!from) {
-            ({ from } = noteValue);
+          if (noteValue.sig) {
+            // try to verify signature
+            const { sig, sigver } = noteValue;
```

Comment:
Move the scope of `sig` and `sigver` to be declared next to `sigValid`

Reply:
resolved